scripts/check_used_functions.py

```diff
@@ -1,24 +1,21 @@
 #!/usr/bin/env python
 """
-Check that all the functions defined in the C API are effectivelly used
+Check that all the functions defined in the C API are being used.
 """
 import os
 import sys
 
+from generated_functions import all_functions
```

Comment:
Same as typed_function, could you put the function in this script ?

Reply:
Done.